bundles/org.openhab.binding.bluetooth.daikinmadoka/src/main/java/org/openhab/binding/bluetooth/daikinmadoka/internal/model/commands/GetCleanFilterIndicatorCommand.java

```diff
@@ -51,7 +54,12 @@ public void handleResponse(Executor executor, ResponseListener listener, MadokaM
         }
 
         setState(State.SUCCEEDED);
-        executor.execute(() -> listener.receivedResponse(this));
+        try {
+            executor.execute(() -> listener.receivedResponse(this));
+        } catch (Exception e) {
```

Comment:
You cannot catch more specific exception(s) ?

Reply:
This binding has numerous calls to `executor.execute(() ->` some of them had a catch block, some not. I checked if any exception was catch higher up the chain and if not add the catch block to make to code more robust/same structured.

I can not determine the possible exception types and more refactoring goes beyond the scope of this PR, if you ask me. But if you can give me some tips, i'm willing to look into it and check if i can improve it. Otherwise please merge as is.

Reply:
The tip would be to check what exception can be thrown by `listener.receivedResponse`

But ok, let's do that in another PR if you like.